cmake/cxx_flags_override.cmake

```diff
@@ -20,6 +20,13 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") OR ("${CMAKE_CXX_COMPILER_ID}"
   set(CMAKE_CXX_FLAGS_MINSIZEREL_INIT "-Os")
   set(CMAKE_CXX_FLAGS_RELEASE_INIT "-O3")
   set(CMAKE_CXX_FLAGS_RELWITHDEBINFO_INIT "-O2 -g")
+
+  if (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
@Mic92 I only just noticed. Why are you setting this in `cmake/cxx_flags_override.cmake`? This is the wrong place for it. This file is meant for overriding the CMake defaults **only**.